srcpkgs/virtualbox-ose/template

```diff
@@ -112,6 +112,8 @@ do_install() {
 
 	# Install modules-load.d file.
 	vinstall ${FILESDIR}/virtualbox.conf 644 usr/lib/modules-load.d
+	vinstall ${FILESDIR}/virtualbox.modprobe 644 \
+		${DESTDIR}/etc/modprobe.d/virtualbox.conf
```

Comment:
```suggestion
	vinstall ${FILESDIR}/virtualbox.modprobe 644 usr/lib/modprobe.d virtualbox.conf
```
package should install to `/usr/lib`, `/etc` is for local admin

also, the syntax for `vinstall` was slightly off

Reply:
Fixed. Tbh I always find `vinstall` syntax a bit weird, original `install -pm644 source target` seems more natural but it's, again, personal.

Reply:
See my comment at https://github.com/void-linux/void-packages/pull/52910#discussion_r1905048888